kmir/src/tests/integration/data/crate-tests/single-bin/single_exe::main.expected

```diff
@@ -5,7 +5,6 @@
 │  (225 steps)
 ├─ 3 (terminal)
 │   #EndProgram ~> .K
-│   function: main
```

Comment:
Have you checked why we are losing this function name?
I would have expected it to be `single_exe::main` now but not disappear; maybe some rendering code has to change accordingly? (not a blocker, though).

Reply:
I couldn't find an obvious fix, opened an issue:
* #681